tests/system/action/projector/test_next.py

```diff
@@ -0,0 +1,53 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class ProjectorNext(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.set_models(
+            {
+                "meeting/1": {},
+                "projector/2": {"meeting_id": 1},
+                "projector/3": {
+                    "current_projection_ids": [1, 2],
```

Comment:
Please add a third stable element and test, that it stays projected

Reply:
Set the second element to stable and test this function.